andrada12/assignment4/WorldMap.java

```diff
@@ -0,0 +1,135 @@
+
+import java.util.ArrayList;
+import java.util.TreeSet;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ *
+ * @author andrada
+ */
+
+ class MapPart{
+        int islandNumber;
+        char type;
+        
+        public MapPart(int islandNumber,  char type){
+            this.islandNumber = islandNumber;
+            this.type = type;
+        }
+        
+        public String toString(){
+            return islandNumber+" ";
+        }
+    }
+
+ class Position{
+        int x;
+        int y;
+        
+        public Position(int x, int y){
+            this.x = x;
+            this.y = y;
+        }
+        
+        public String toString(){
+            return x + " " +y;
+        }
+    }
+
+public class WorldMap {
+    
+   
+    
+    /* Calculates the positions of the neighbours of a given node.
+        Returns just the neighbours that are land*/
```

Comment:
This is decent code. Sorry I have left it too late to give it a good review. I think the one thing that could be improved is simplification for readability. I'll talk about this is the group session